core/src/main/java/org/sonarsource/sonarlint/core/plugin/PluginClassloaderFactory.java

```diff
@@ -51,7 +50,6 @@
   private static final String API_CLASSLOADER_KEY = "_api_";
 
   private final TempFolder temp;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "temp" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)